### PR TITLE
BZ2024348: Add bare metal to the requirements for installing on a SNO

### PR DESCRIPTION
**Pull request by ogradyp**
Applies only to the Enterprise-4.9 branch & main

Browse the Bugzilla here:
https://bugzilla.redhat.com/show_bug.cgi?id=2024348

Browse the preview update here:
https://deploy-preview-39934--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_sno/install-sno-preparing-to-install-sno

### CODE REVIEW

File: modules/install-sno-requirements-for-installing-on-a-single-node.adoc [hunk 1]
```diff
@@ -9,6 +9,8 @@ Installing {product-title} on a single node alleviates some of the requirements
 
 * *Administration host:* You must have a computer to prepare the ISO, to create the USB boot drive, and to monitor the installation.
 
+* *Bare metal installation:* Installing {product-title} on a single node on bare metal requires you specify the `platform.none: {}` parameter in the `install-config.yaml` configuration file.
```
Comment (mburke5678) [inline modules/install-sno-requirements-for-installing-on-a-single-node.adoc:12]: s/requires/requires that

**Comment by mburke5678** [comment]
@ogradyp One small nit. Otherwise LGTM

File: modules/install-sno-requirements-for-installing-on-a-single-node.adoc (hunk 1 above)
Comment (ogradyp) [reply modules/install-sno-requirements-for-installing-on-a-single-node.adoc]: Thanks for your feedback Michael. I have applied your suggestion. It now reads...

Bare metal installation: Installing OpenShift Container Platform on a single node on bare metal requires that you specify the platform.none: {} parameter in the install-config.yaml configuration file.

**Pull request merged**